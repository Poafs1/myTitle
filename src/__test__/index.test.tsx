import React from 'react'
import SayHello from '../index'
import { shallow } from 'enzyme'

describe("Say hello to me", () => {

  it("Should return hello sentence include my name", () => {
    const wrapper = shallow(<SayHello name="Poafs1"/>)
    expect(wrapper).toMatchSnapshot();
  })

})