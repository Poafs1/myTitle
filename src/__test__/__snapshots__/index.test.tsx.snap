// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Say hello to me Should return hello sentence include my name 1`] = `
<div>
  <p>
    Hey 
    Poafs1
    , say hello to TypeScript.
  </p>
</div>
`;
